aibolit/__main__.py

```diff
@@ -486,15 +492,21 @@ def get_exit_code(results):
     files_analyzed = len(results)
     errors_number = 0
     perfect_code_number = 0
+    errors_strings = []
     for result_for_file in results:
         results = result_for_file.get('results')
         errors_string = result_for_file.get('error_string')
         if not results and not errors_string:
             perfect_code_number += 1
         elif not results and errors_string:
-            errors_number += 1
+            if 'javalang.parser.JavaSyntaxError' not in errors_string:
```

Comment:
Is it possible to match Exception by type not by strings?

Reply:
no, because we keep only string here